css/index.css

```diff
@@ -1,3 +1,370 @@
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+	margin: 0 auto;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+	
+	margin-bottom: 15px;
+	font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+}
+
+
+
+
+
+/* Your code starts here! */
+
+/* ****************************************************************
+*                        CONTAINER                                *
+*******************************************************************/ 
+.container {
+	max-width: 880px;
+	margin: 0 auto;
+}
+
+@media (max-width: 800px) {
+	.container {
+		margin: 0 10px;
+	}
+}
+
+@media (max-width: 500px) {
+	.container {
+		margin: 0;
+	}
+}
+
+/* ***************************************************************
+*                          HEADER                                *
+*****************************************************************/
+header {
+	display: flex;
+}
+
+.logo-link {
+	margin-top: 22px;
+}
+
+	/***************  MEDIA QUERY 800PX  ***************/
+	@media (max-width: 800px) {
+		header {
+			flex-direction: column-reverse;
+		}
+
+		.logo-link {
+			align-self: center;
+			margin-top: 30px;
+			margin-left: 13px;
+		}
+	}
+
+	/***************  MEDIA QUERY 800PX  ***************/
+	@media (max-width: 500px) {
+		.logo-link {
+			margin-top: 19px;
+		}
+	}
+
+/* ***************************************************************
+*                            NAV                                 *
+*****************************************************************/
+nav {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	margin-top: 34px;
+}
+
+nav > a {	
+	font-size: 16px;
+	color: #B3B3B3;
+	text-decoration: none;
+}
+
+nav > a:last-of-type {
+	margin-right: 90px;
+}
+
+	/***************  MEDIA QUERY 800PX  ***************/
+	@media (max-width: 800px) {
+		nav {
+			justify-content: space-between;
+			margin-top: 40px;
+			padding: 0 20px;
+		}
+		
+		nav > a:last-of-type {
+			margin-right: 0;
+		}
+	}
+
+	/***************  MEDIA QUERY 500PX  ***************/
+	@media (max-width: 500px) {
+		nav {
+			flex-direction: column;
+			justify-content: center;
+			margin-top: 20px;
+			padding: 0;
+		}
+
+		nav a {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+
+			width: 100%;
+			height: 61px;
+			border-top: 1px solid #808080;
+			
+			font-size: 20px;
+		}
+
+		nav a:last-of-type {
+			border-bottom: 1px solid #808080;
+		}
+	}
+
+/* **************************************************************
+*                          BANNER                               *
+****************************************************************/
+.banner-container {
+	display: flex;
+	align-items: center;
+	padding-top: 72px;
+	padding-bottom: 75px;
+	border-bottom: 2px solid black;
+}
+
+.slogan-container {
+	margin-left: 127px;
+    margin-right: 0px;
```

Comment:
We have some uneven spacing here, which hurts readability. You're a VS Code user, right? Remind me to show you the ways of "prettier", so you can fix formatting things like this with one command.